packages/cli/src/__tests__/commands/bundle.test.ts

```diff
@@ -24,7 +25,7 @@ describe('bundle', () => {
   afterEach(() => {
     (lint as jest.Mock).mockClear();
     (bundle as jest.Mock).mockClear();
-    (getTotals as jest.Mock).mockClear();
+    (getTotals as jest.Mock).mockReset();
```

Comment:
Why do we reset instead of clear? Do we maybe want to do the reset on the previous two as well?

Reply:
With `mockReset` here we pass the exit code from one test to the next one.

Reply:
`mockReset` does everything that [mockFn.mockClear()](https://jestjs.io/docs/mock-function-api#mockfnmockclear) does, and also removes any mocked return values or implementations.

Reply:
> With mockReset here we pass the exit code from one test to the next one.

Looks like you want to do the opposite: to NOT pass the exit code through tests 🙂 Am I not correct?

Reply:
Yes, you're right.